src/Apps/Auction/Components/AuctionDetails/AuctionInfoSidebar.tsx

```diff
@@ -16,17 +16,17 @@ const AuctionInfoSidebar: React.FC<AuctionInfoSidebarProps> = ({ sale }) => {
       <Box>
         <Text variant="sm">Questions?</Text>
 
-        <RouterLink to="/how-auctions-work" target="_blank">
+        <Link as={RouterLink} href="/how-auctions-work" target="_blank">
```

Comment:
This shouldn't be a `RouterLink` since it's just opening in a new tab

Reply:
Ah yes I didn't consider the `target="_blank"` I will update these ones